tests/10-vcdm2.js

```diff
@@ -310,6 +338,7 @@ describe('Verifiable Credentials Data Model v2.0', function() {
         await assert.rejects(endpoints.issue(require(
           './input/credential-issuer-name-extra-prop-en-fail.json')));
       });
+      // TODO: remove. no longer present.
```

Comment:
looks like the title just changed: https://w3c.github.io/vc-data-model/#names-and-descriptions